alicloud/resource_alicloud_instance_test.go

```diff
@@ -812,6 +812,33 @@ func TestAccAlicloudInstanceType_update(t *testing.T) {
 	})
 }
 
+func TestAccAlicloudInstanceTypePrepaid_update(t *testing.T) {
+	var instance ecs.Instance
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckInstanceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckInstanceTypePrepaid(EcsInstanceCommonTestCase),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckInstanceExists("alicloud_instance.type", &instance),
+				),
+			},
+
+			{
+				Config: testAccCheckInstanceTypePrepaidUpdate(EcsInstanceCommonTestCase),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckInstanceExists("alicloud_instance.type", &instance),
+				),
+			},
```

Comment:
Please modify the instance's charge type to `PostPaid` to ensure it can be removed.

Reply:
I have set force_delete options to delete PrePaid instance